### PR TITLE
Release v1.20.0

### PR DESCRIPTION
**Pull request by amantoux**

### CODE REVIEW

File: packages/fleather/README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Add Fleather to your dependencies.
 dependencies:
   flutter:
     sdk: flutter
-  fleather: ^1.19.0
+  fleather: ^1.12.0
```
Comment (Amir-P) [inline packages/fleather/README.md:31]: I think it was meant to be `1.20.0`.

**Review by Amir-P — CHANGES_REQUESTED** [review]
Thanks, Alan. I believe we should also have a new release for Parchment as well since we had changes for Flutter 3.27. I also left a tiny comment.

**Comment by amantoux** [comment]
@Amir-P Can you please review changes?

**Review by Amir-P — CHANGES_REQUESTED** [review]
One more thing, Alan. We also need to update the parchment version in `/fleather/pubspec.yaml` file.

**Review by Amir-P — APPROVED** [review]
LGTM!

**Pull request merged**